### DISCUSSION TITLE
Adding validation field in custom plugin

### DISCUSSION THREAD

**Discussion by BellBoy68** [Budibase/budibase#10300] [Help] [answered]
Hi everyone. I installed this custom plugin in my database: 
[https://github.com/melohagan/budibase-component-nikita-input-mask](url)
But I can't do validation for this field. Can I add validation to this plugin? If yes, how to do it?
Thank you in advance.

**Comment by andz-bb** [comment, marked as answer]
hey @BellBoy68

Thanks for the suggestion! I've just added an update with that to Mel's component now. Check his repo for the updated version 1.1.2 which has the validation settings included. Hope it works well for you! 😊

**Reply by BellBoy68** [reply]
Yes. Everything works in the web version. Could you tell me how the .tar.gz file is built in order to install the plugin locally?

**Reply by ConorWebb96** [reply]
Hey @BellBoy68,

You can download it from here: https://github.com/melohagan/budibase-component-nikita-input-mask/releases/tag/v1.1.2, it should be near the bottom of the page!

Hopefully this helps!

**Reply by BellBoy68** [reply]
Thank you very much!
